guides/manual-installation.md

```diff
@@ -1,19 +1,20 @@
+# Installation of Trento
+
 ## Scope
 
-This documents covers the step to manually install trento server without relying on the trento helm chart or kubernetes all together. The lastest
-available version of SUSE Linux Enterprise Server for SAP Applications is used as the base operating system, which is 15 SP5 at the time of writing.
+This document covers the steps to manually install trento server without relying on the trento helm chart or kubernetes all together. The latest available version of SUSE Linux Enterprise Server for SAP Applications is used as the base operating system, which is 15 SP5 at the time of writing.
```

Comment:
I would remove any mention to the the helm chart. Having that way first is just a coincidence.
Just `This document covers the steps to manually install Trento server` would be enough in my opinion